modules/mailer/mailer.go

```diff
@@ -58,7 +58,7 @@ func NewMessageFrom(to []string, from, subject, body string) *Message {
 
 // NewMessage creates new mail message object with default From header.
 func NewMessage(to []string, subject, body string) *Message {
-	return NewMessageFrom(to, setting.MailService.From, subject, body)
+	return NewMessageFrom(to, setting.MailService.From, "", subject, body)
```

Comment:
But as stated in https://github.com/go-gitea/gitea/blob/master/conf/app.ini#L312 From can be in format `"Sender" <sender@domain.ext>` so it should be parsed somehow and passed in both arguments